shared/spotify-api/src/mockClient.ts

```diff
@@ -354,6 +373,7 @@ export class MockSpotifyClient implements SpotifyDataSource {
       uri: 'https://discord.gg/graphos',
       product: 'free',
       type: 'user',
+      policies: {},
```

Comment:
I don't see this field listed in the [`/me` endpoint](https://developer.spotify.com/documentation/web-api/reference/get-current-users-profile) in the developer documentation, nor do we have a field defined in the [GraphQL schema](https://github.com/apollographql/spotify-showcase/blob/71475e336f7fbf45e54f4bc337661adddd1b4201/subgraphs/spotify/schema.graphql#L893-L949) for this. Is this needed?

Reply:
When I regenerated the mocks, there were a few fields (this included) that appeared—I assume they came from the API, so I made their types super generic since I didn't want to go hunting the docs, and also since we're not surfacing them yet 😛.

Reply:
Discussed over slack. Sounds like this is likely due to some extra "publisher" permissions on the token used to generate the mocks from the API. Since 99% of the users of this app will be typical consumers, we'd like to err on the side of less exposed data.